specs-go/config.go

```diff
@@ -117,6 +117,10 @@ type Mount struct {
 	Source string `json:"source,omitempty"`
 	// Options are fstab style mount options.
 	Options []string `json:"options,omitempty"`
+
+	// UID/GID mappings used for changing file owners w/o calling chown, fs should support it
+	// every mount point could have its own mapping
+	IDMappings []LinuxIDMapping `json:"id_mappings,omitempty"`
```

Comment:
One thing I don't understand is why we do not have separate UID and GID mappings, can you please explain?

Reply:
I think I kept in mind one limited case, I agree in for general purpose, better to have both uid and gid.